### ISSUE TITLE
Documentation out of date?

### ISSUE THREAD

**Issue by bsabiston** [googlevr/gvr-ios-sdk#314]
I'm trying to use the SDK and looking at the TreasureHunt code walkthrough here: https://developers.google.com/vr/develop/ios/get-started

But the code does not match the actual sample project.  For example, instead of allocating a GVRCardboardView in loadView(), the current sample code creates a GVRRendererViewController in viewDidLoad().

I already have a custom view controller so I am trying to do it the old way, and in Swift, but I am getting an error on the initialization of GVRCardboardView:

cardboardView = GVRCardboardView(initWithFrame:CGRectZero)

gives this error: Argument labels '(initWithFrame:)' do not match any available overloads
It looks to me like GVRCardboardView's designated initializer does match my call, so I don't understand the error.  Is it not recommended to do it this way now and do I need to use this GVRRendererViewController instead?

**Comment by Performador** [comment]
We currently do not have plans for a Swift version. Is there a particular reason you're looking for a Swift framework?

**Comment by bsabiston** [comment]
Just seems like Objective C is on the way out.  Swift seems to be the thing to be using for current Apple software.

**Comment by Performador** [comment]
You are correct but under the hood, most of GVRSDK is C++ with minimal Obj-C which you should be able to import into Swift land (except for #312 which we're looking into). I do not expect a pure Swift implementation to be any more performant.

**Comment by bsabiston** [comment]
Okay.  What I really need is a Metal version.  I'm trying to Frankenstein the SDK onto my Metal app currently.  It's kind of working.  Is the source code available for the SDK?  I'm trying to figure out how to do the lens distortion...

**Comment by Performador** [comment]
Unfortunately the mesh distortion code is not open source. We're looking into how we can support Metal.

**Comment by bsabiston** [comment]
Okay, thanks.  I'm thinking now that instead of using Metal to display the scene, I should probably just use it to render to a texture and then pass that back to the google cardboardView so that it can do the lens distortion.

**Comment by NeethuAntony** [comment]
Hi did the gvr-ios-sdk is available which uses Metal Api for rendering?
